wvanreeven/all_sky.ipynb

```diff
@@ -0,0 +1,87 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsst.ts import salobj\n",
```

Comment:
You may want to consider splitting this into multiple cells. I'd at least put the import statement in the first cell, one cell for the domain, one cell for the remote, one cell for the start_task. Everything else can go in the same cell.

Reply:
Thanks for the suggestion. I put them all together because to me it is boilerplate code, i.e. necessary code to get the CSC started after which the interesting stuff happens. Could you explain why you advise me to split this cell up into mulitple cells? Perhaps we can discuss this in Slack. I'll ask the same question to you there.